docs/Configuring-Orion/TLS.md

```diff
@@ -3,15 +3,162 @@ description: TLS
 
 # TLS
 
-Orion supports the TLS (Transport Layer Security) protocol to enable secure communications.
+Orion supports the Transport Layer Security (TLS) protocol to enable secure communications. TLS is used between Orion nodes only. It is not used between Pantheon and Orion nodes.
 
 Enable TLS by setting the `tls` property to `strict` in the [Orion configuration file](Configuration-File.md).  
 
 ## Getting Started with TLS
 
 To have Orion generate certificates, start the nodes with the TLS trust mode set to `insecure-no-validation` for `tlsclienttrust` 
 and `tlsservertrust` until the `tlsknownclients` and `tlsknownservers` files are populated. When the files are populated, 
-restart the nodes with the TLS trust mode set to `whitelist` for `tlsclienttrust` and `tlsservertrust`. 
+restart the nodes with the TLS trust mode set to `whitelist` for `tlsclienttrust` and `tlsservertrust`.
+
+The following procedures explain how to generate certificates when the Common Name (CN) is either the public DNS or an IP address by using [OpenSSL](https://www.openssl.org/source/). Before you begin, ensure OpenSSL is installed.
+
+Choose one of the following methods:
+
+* [Generating certificates when the public DNS is the CN](#Generating-certificates-when-the-public-DNS-is-the-CN)
+* [Generating certificates when the public IP address is the CN](#Generating-certificates-when-the-public-IP-address-is-the-CN)
+
+### Generating certificates when the public DNS is the CN
+
+If you want to use a public DNS as the CN, complete the following steps:
+
+#### Generating a CA certificate
+
+1. Generate a key file called orion_ca.key by running the following command:
```

Comment:
Generate a key file called orion_ca.key by running the following command: -> Generate a key file called orion_ca.key:

Same reason as above.  

Also need code style for the filename as per our doc standards (ie, `orion_ca.key`)

Reply:
CA needs to be included in full the first time it's used.  @lucassaldanha -> what is the acronym for?

Reply:
certificate authority (https://en.wikipedia.org/wiki/Certificate_authority)